packages/project-management-automation/lib/tasks/first-time-contributor-label/index.js

```diff
@@ -14,12 +14,20 @@ const debug = require( '../../debug' );
  */
 async function firstTimeContributorLabel( payload, octokit ) {
 	const userType = payload.pull_request.user.type;
+	const authorAssociation = payload.pull_request.author_association;
 
 	if ( userType === 'Bot' ) {
 		debug( 'first-time-contributor: User is a bot. Aborting' );
 		return;
 	}
 
+	if ( authorAssociation !== 'FIRST_TIME_CONTRIBUTOR' ) {
```

Comment:
I think something like this might be a good idea, just in case github changes their API and removes `author_association`
```suggestion
	if ( authorAssociation && authorAssociation !== 'FIRST_TIME_CONTRIBUTOR' ) {
```

Reply:
Good observation